GroundDBMemoryPool/src/rdma/util.cc

```diff
@@ -321,42 +324,13 @@ int post_receive(const struct resources *res)
  * This function creates and allocates all necessary system resources. These
  * are stored in res.
  *****************************************************************************/
-int resources_create(struct resources *res, const char *serverName, const int tcpPort, const char *devName, const int ibPort)
+int resources_create(struct resources *res, const char *serverName, const char *devName, const int ibPort)
 {
     struct ibv_device **dev_list = NULL;
-    struct ibv_qp_init_attr qp_init_attr;
     struct ibv_device *ib_dev = NULL;
-    size_t size;
     int i;
-    int mr_flags = 0;
-    int cq_size = 0;
     int num_devices;
     int rc = 0;
-    /* if client side */
-    if (serverName)
-    {
-        res->sock = sock_connect(serverName, tcpPort);
-        if (res->sock < 0)
-        {
-            fprintf(stderr, "failed to establish TCP connection to server %s, port %d\n",
```

Comment:
Unsure if it is needed anymore. If not, then `sock_connect()` is unused and can be removed.

Reply:
I have moved this part of code into `connect_qp`